openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -58,35 +59,35 @@ def get_completed_programs(student):
         student (User): Representing the student whose completed programs to check for.
 
     Returns:
-        list of program ids
+        list of program UUIDs
 
     """
-    meter = ProgramProgressMeter(student)
 
+    meter = ProgramProgressMeter(student, use_catalog=True)
     return meter.completed_programs
 
 
 def get_awarded_certificate_programs(student):
     """
-    Find the ids of all the programs for which the student has already been awarded
+    Find the UUIDs of all the programs for which the student has already been awarded
     a certificate.
 
     Args:
         student:
             User object representing the student
 
     Returns:
-        ids of the programs for which the student has been awarded a certificate
+        UUIDs of the programs for which the student has been awarded a certificate
 
     """
-    return [
-        credential['credential']['program_id']
-        for credential in get_user_credentials(student)
-        if 'program_id' in credential['credential'] and credential['status'] == 'awarded'
-    ]
+    certificate_programs = []
+    for credential in get_user_credentials(student):
+        if 'program_uuid' in credential['credential'] and credential['status'] == 'awarded':
+            certificate_programs.append(credential['credential']['program_uuid'])
```

Comment:
@MatthewPiatetsky cool, I'm just making sure it's actually going to be addressed by @clintonb's work. You know what happens when you assume...